### PR TITLE
add references to third party tools and libraries to docs

### PR DESCRIPTION
**Pull request by coreyoconnor**
adds references to `scala-java-time`, `scala-java-locale` and a few performance tools.

### CODE REVIEW

**Comment by ekrich** [comment]
Would like some business for ekrich/sjavatime too. It is missing some stuff but could be improved with a little TLC.

**Comment by coreyoconnor** [comment]
> ekrich/sjavatime

Added!

File: docs/user/profiling.md [hunk 1]
```diff
@@ -53,6 +56,19 @@ Exit status: 0
 A [flamegraph](http://www.brendangregg.com/flamegraphs.html) is a
 visualization of the most frequent code-paths of a program. You can use
 flamegraphs to see where your program spends most of its CPU time.
+
+### Use samply
+
+[samply](https://github.com/mstange/samply) is a command line CPU profiler which uses the Firefox profiler as
+its UI. Samply has support for de-mangling Scala native symbols.
```
Comment (keynmol) [inline docs/user/profiling.md:63]:
```suggestion
its UI. Samply has support for de-mangling Scala Native symbols.
```

File: docs/user/profiling.md [hunk 2]
```diff
@@ -53,6 +56,19 @@ Exit status: 0
...
+
+### Use hotspot
+
+[hotspot](https://github.com/KDAB/hotspot) is a GUI for `perf` which provides a flamegraph view. As of this
+writing, hotspot does not de-mangle Scala native symbols.
```
Comment (keynmol) [inline docs/user/profiling.md:68]:
```suggestion
writing, hotspot does not de-mangle Scala Native symbols.
```

**Review by WojciechMazur — APPROVED** [review]

**Pull request merged**